apps/docs/src/development/grid.md

```diff
@@ -0,0 +1,30 @@
+# Grid System
+
+::: warning Work in progress / Active development
+This library is currently in early / active development.
+:::
+
+The grid implementation in **onyx** differs from more popular grid system implementation, that you might know.
```

Comment:
```suggestion
The grid implementation in **onyx** differs from other design systems that you might know.
```

we should not downgrade ourselves by saying we are unpopular^^

Reply:
I would see it as being humble^^